src/Components/DataPath.php

```diff
@@ -132,6 +132,10 @@ protected function filterMimeType($mimetype)
             throw new InvalidArgumentException(sprintf('invalid mimeType, `%s`', $mimetype));
         }
 
+        if ($mimetype === 'text/vcard') {
```

Comment:
Well according to [RFC6350](https://tools.ietf.org/html/rfc6350#section-3) the correct mimetype is `text/vcard` so maybe updating the test suite to be less strict accepting both mimetype depending on FileInfo data is better than bugfixing something which is out of scope for this PR